tests/test_operators.py

```diff
@@ -386,3 +387,36 @@ def test_length_balancer(self):
             targets=targets,
             tester=self,
         )
+
+    def test_augment_whitespace_model_input(self):
+        source = "The dog ate my cat"
+        inputs = [{"source": source}]
+
+        operator = AugmentWhitespace(augment_model_input=True)
+        outputs = apply_operator(operator, inputs)
+        assert outputs[0]["source"] != source, f"Source of f{outputs} is equal to f{source} and was not augmented"
+        normalized_output_source = outputs[0]["source"].split()
+        normalized_input_source = source.split()
+        assert (
+            normalized_output_source == normalized_input_source
+        ), f"{normalized_output_source} is not equal to f{normalized_input_source}"
+
+    def test_augment_whitespace_task_input_with_error(self):
+        text = "The dog ate my cat"
+        inputs = [{"inputs": {"text": text}}]
+        operator = AugmentWhitespace(augment_task_input=True)
+        operator.set_task_input_fields(["sentence"])
+        with self.assertRaises(ValueError):
+            outputs = apply_operator(operator, inputs)
+
+    def test_augment_whitespace_task_input(self):
```

Comment:
add a test that its really only affecting the input and not the rest

Reply:
There is such a test.  But it requires the standard recipe.  So it's in test_recipe.py:

  def test_standard_recipe_with_augmentor_on_task_input(self):
        recipe = StandardRecipeWithIndexes(
            card="cards.sst2",
            augmentor="augmentors.augment_whitespace_task_input",
            template_card_index=0,
            max_train_instances=0,
            max_test_instances=2,
        )
        stream = recipe()
        sample = list(stream["test"])[1]
        source = sample["source"]
        pattern = "Given this sentence: (.*), classify if it is negative, positive.\s*"
        result = re.match(pattern, sample["source"], re.DOTALL)
        assert result, f"Unable to find '{pattern}' in '{source}'"
        result = result.group(1)
        original_text = "unflinchingly bleak and desperate "
        assert (
            result != original_text
        ), f"Augmented text '{result}' is equal to text without '{original_text}' and was not augmented"
        normalized_output_source = result.split()
        normalized_input_source = original_text.split()
        assert (
            normalized_output_source == normalized_input_source
        ), f"{normalized_output_source} is not equal to f{normalized_input_source}"